src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days <= 2) {
+    return days * 40;
+  }
+
+  if (days > 2 && days < 7) {
+    return (days * 40) - 20;
```

Comment:
use DRY approach. to avoid repeating code create variable x = days*40 and use it further.